src/checkovInstaller.ts

```diff
@@ -12,44 +15,145 @@ const isCheckovInstalledGlobally = async () => {
     }
 };
 
-const installOrUpdateCheckovWithPip3 = async (logger: Logger): Promise<string | null> => {
+const validateCheckovVersion = (checkovVersion: string | undefined) => {
+    // validates that the version is >= the min version (or it can be null)
+    // does NOT validate that the version actually exists (this will happen during the actual installation)
+    // assumes the value came from getCheckovVersion helper method
+    // a normal return indicates the version is valid, otherwise an error is thrown
+
+    if (!checkovVersion) {
+        return;
+    }
+
+    const parts = checkovVersion.split('.').map(i => parseInt(i));
+    if (parts.length !== 3) {
+        throw new Error(`Invalid specified checkov version: ${checkovVersion} (expected format #.#.#)`);
+    }
+
+    const minVersion = minCheckovVersion.split('.').map(i => parseInt(i));
```

Comment:
There is an awesome package that does all of these, I recommend using it instead of implementing it on our own 
https://www.npmjs.com/package/semver